_pytest/deprecated.py

```diff
@@ -22,3 +27,6 @@
 GETFUNCARGVALUE = "use of getfuncargvalue is deprecated, use getfixturevalue"
 
 RESULT_LOG = '--result-log is deprecated and scheduled for removal in pytest 4.0'
+
+MARK_INFO_ATTRIBUTE = RemovedInPytest4_0Warning(
+    "Markinfo attributes are deprecated, please iterate the mark Collection")
```

Comment:
What do you mean? Actually, now that I read it again, I'm not sure what you mean by "please iterate the mark Collection"... oh do you mean the marks in the *collected* item?